asm/non_matchings/overlays/actors/ovl_Bg_Jya_Cobra/BgJyaCobra_UpdateShadow_Side.s

```diff
@@ -82,10 +82,10 @@ glabel func_80895EF0
 /* 006F8 80895FE8 4481B000 */  mtc1    $at, $f22                  ## $f22 = 0.50
 /* 006FC 80895FEC 3C018089 */  lui     $at, %hi(D_8089772C)       ## $at = 80890000
 /* 00700 80895FF0 3C148089 */  lui     $s4, %hi(D_808973A4)       ## $s4 = 80890000
-/* 00704 80895FF4 3C178089 */  lui     $s7, %hi(D_808973B0)       ## $s7 = 80890000
+/* 00704 80895FF4 3C178089 */  lui     $s7, %hi(D_808973A4)       ## $s7 = 80890000
 /* 00708 80895FF8 3C108089 */  lui     $s0, %hi(D_8089731C)       ## $s0 = 80890000
 /* 0070C 80895FFC 2610731C */  addiu   $s0, $s0, %lo(D_8089731C)  ## $s0 = 8089731C
-/* 00710 80896000 26F773B0 */  addiu   $s7, $s7, %lo(D_808973B0)  ## $s7 = 808973B0
+/* 00710 80896000 26F773B0 */  addiu   $s7, $s7, %lo(D_808973A4) + 0xC  ## $s7 = 808973B0
```

Comment:
This hi/lo pair one also needs to be fixed for the same reasons I mentioned in my previous review comments